redisai/model_test.go

```diff
@@ -10,23 +10,27 @@ func Test_modelGetParseReply(t *testing.T) {
 		reply interface{}
 	}
 	tests := []struct {
-		name        string
-		args        args
-		wantBackend string
-		wantDevice  string
-		wantTag     string
-		wantBlob    []byte
-		wantErr     bool
+		name             string
+		args             args
+		wantBackend      string
+		wantDevice       string
+		wantTag          string
+		wantBlob         []byte
+		wantBatchsize    int64
```

Comment:
@AvitalFineRedis can we extend these tests to include tag, batchsize, inputs, outputs, minbatchsize info?